src/Microsoft.FeatureManagement/FeatureVariant.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.FeatureManagement
+{
+    /// <summary>
+    /// A variant of a feature.
+    /// </summary>
+    public class FeatureVariant
```

Comment:
Just to get your thoughts... if we follow the naming convention of filters, should this be called `FeatureVariantConfiguration`?

Reply:
I am leaning towards keeping it as FeatureVariant. I feel that this is a bit different from what we have here for feature filters.

A feature filter is the code component that allows conditional enablement of a feature flag. So the objects that we call `FeatureFilterConfiguration` really aren't the feature filters themselves, they are the configuration to setup up the evaluation of a feature filter.

On the other hand, the `FeatureVariant` we have here is not setting up some other component. I would say that this is the variant itself.

Reply:
Agreed. Makes sense.